### ISSUE TITLE
Potential path parsing issue with Windows

### ISSUE THREAD

**Issue by discdiver** [PrefectHQ/prefect-dbt#70]
Potential Windows path parsing issue report from community user with prefect-dbt 

https://linen.prefect.io/t/2712145/

**Comment by ahuang11** [comment]
Also https://prefect-community.slack.com/archives/CL09KU1K7/p1664567323759169?thread_ts=1664418745.917639&cid=CL09KU1K7

**Comment by costructuralist** [comment]
Potentially related:

Running the following flow locally on Windows (from within a Poetry environment):

```
@flow
def dbt_flow(dbt_project_path):
    trigger_dbt_cli_command(command="dbt test", project_dir="""C:\Source\data\debut""")
```
gives the following error:
```
16:15:35.503 | INFO    | Task run 'trigger_dbt_cli_command-321ca940-0' - Running dbt command: dbt test --profiles-dir C:\Users\myusername\.dbt --project-dir C:\Source\data\debut
16:15:35.775 | ERROR   | Task run 'trigger_dbt_cli_command-321ca940-0' - Encountered exception during execution:
Traceback (most recent call last):
  File "C:\Users\myusername\AppData\Local\pypoetry\Cache\virtualenvs\prefect-etl-szksG6RR-py3.10\lib\site-packages\prefect\engine.py", line 1183, in orchestrate_task_run
    result = await task.fn(*args, **kwargs)
  File "C:\Users\myusername\AppData\Local\pypoetry\Cache\virtualenvs\prefect-etl-szksG6RR-py3.10\lib\site-packages\prefect_dbt\cli\commands.py", line 158, in trigger_dbt_cli_command
    result = await shell_run_command.fn(command=command, **shell_run_command_kwargs)
  File "C:\Users\myusername\AppData\Local\pypoetry\Cache\virtualenvs\prefect-etl-szksG6RR-py3.10\lib\site-packages\prefect_shell\commands.py", line 96, in shell_run_command
    raise RuntimeError(msg)
RuntimeError: Command failed with exit code 127:
/bin/bash: C:UsersmyusernameAppDataLocalTempprefect-atmv4tpr: No such file or directory
```

Looking at the last line, the path slashes have been removed.

**Comment by ahuang11** [comment]
Maybe can use this:
https://github.com/PrefectHQ/prefect/pull/7261

**Comment by discdiver** [comment]
Closing, as I believe this was fixed by prefect-shell v0.1.2, based on the Slack/Linen thread referenced above.
